development/src/components/welcome_comp.js

```diff
@@ -2,48 +2,14 @@ import Onboarding from "./onboarding_comp";
 import Image from "next/image";
 import welcome from "../../public/assets/dashboard/welcome_comp.svg";
 import { Modal } from "./modal";
-import SignUpComponent from "./signup_comp";
 import { usePathname, useRouter, useSearchParams } from "next/navigation";
 import { useEffect } from "react";
-import UserLoginComp from "./userLogin_comp";
-import { ForgotPassword } from "./ForgotPassword";
 
 const Welcome = () => {
   const router = useRouter();
   const view = useSearchParams().get("view");
   return (
     <div>
-      {view == "login" ? (
```

Comment:
Why is the code through line 45 being taken out?

Reply:
I removed those lines of codes from the welcome component because when it is closed, like when a user is on a different page and wants to login or signup through the buttons at the top right, it does not route to either of the components I removed, it only routes when on the welcome page. They were moved to page.js file and that made those components accessible regardless of whatever page the user is on. I hope this answers your question.